test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -1109,6 +1151,12 @@ UCS_TEST_P(test_uct_sockaddr_stress, many_clients_to_one_server)
     /* destroy all the eps here (and not in the test's destruction flow) so that
      * no disconnect callbacks are invoked after the test ends */
     m_entities.clear();
+
+    /* destroyed EPs don't invoke CBs, need to clean up user data manually */
+    ucs::scoped_mutex_lock lock(m_ep_sa_data_lock);
+    while(!m_ep_sa_data.empty()) {
+        ASSERT_TRUE(del_user_data(m_ep_sa_data.begin()->first));
+    }
```

Comment:
1. space after while
2. add helper function "release_user_data()" or "free_user_data()" for this block?